<?xml version="1.0"?>
<launch>
  <arg name="config" default="real_params.yaml"/>
  <arg name="map" default="/costmap_node/costmap"/>
  <arg name="velocity_file" default="velocityset"/>
        
  <node name="siar_planner_action_server" pkg="siar_planner" type="siar_planner_node" output="screen" >
    <remap from="/altitude_map" to="$(arg map)"/>
    <rosparam file="$(find siar_planner)/test/$(arg config)" command="load"  /> 
    <param name="velocity_file" value="$(find siar_planner)/test/$(arg velocity_file)" />
  </node>
  
</launch>
